Makefile

```diff
@@ -0,0 +1,32 @@
+# Makefile for etcdutil
+
+## Dependency versions
+ETCD_VER=v3.3.10
+
+## DON'T EDIT BELOW THIS LINE
```

Comment:
This needs a reason why we DO NOT edit it.

Reply:
Thank you. 
There is no good reason to prohibit editing the lines below the comment.
So, I would delete the comment.